modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,88 @@
+[server-major-upgrade]
+= Server - Major Upgrade
+
+From time to time, {productname} Server upgrades are labeled as major.
+This is especially the case, if a version upgrade of openSUSE Leap is needed. openSUSE Leap is the underlying base operating system (OS).
+For background information about upgrading openSUSE Leap, see https://doc.opensuse.org/documentation/leap/startup/single-html/book-opensuse-startup/index.html#sec-update-zypper.
+
+Run the [command]``/usr/lib/susemanager/bin/server-migrator.sh`` script to upgrade the base OS and {productname} Server.
+
+[NOTE]
+====
+If the [command]``server-migrator.sh`` script is not already installed on your {productname} Server system, update the [package]``susemanager`` package:
```

Comment:
The script is always installed since Uyuni 4.0.2. But you *always* need to update `susemanager` before running the script!

Reply:
ok, change.  Maybe, I'd better recommend to do general package update (maintenance resp. online update first).

Reply:
No, don't do that. During such upgrades you need to update `susemanager` first, then run the script. In that precise order.

That is because the script will change your repositories (to use the new Leap version), will perform an upgrade with vendor change allowed and (not this time) will fix any packages incompatibilities for you.